### ISSUE TITLE
Reading system accessibility

### ISSUE THREAD

**Issue by mattgarrish** [w3c/epub-specs#1608]
The appendix on reading system accessibility was removed from the accessibility specification during the iso process, but the reading system specification still references it:

> It SHOULD meet the accessibility requirements for Reading Systems in [EPUB-A11Y-10].

Should we restore this appendix, or how else can we address this?

**Comment by GeorgeKerscher** [comment]
IMO it is important to keep Reading System accessibility in the appendix. People often confuse document accessibility, software accessibility, and the importance of having both.

**Comment by murata2makoto** [comment]
I prefer a separate specification.   This is what WAI does.

**Comment by mattgarrish** [comment]
> I prefer a separate specification.

That's not in scope or realistic for this revision. We need something we can use now.

Given that [there are only two bullets](http://idpf.org/epub/a11y/#app-ac-rs-conf) for reading systems, could we integrate these directly in the reading system specification rather than farm them out to the accessibility specification?

**Comment by avneeshsingh** [comment]
Separate specifications for reading systems accessibility is difficult to fit in the current schedule. We need to wrap up before November 2021 (CR timeline). 
As per my understanding, the purpose of mentioning reading systems accessibility in the appendix of EPUB accessibility is to make it clear that EPUB accessibility spec is focused on the content, and one need to focus separately on accessibility of reading systems and to point to couple of resources for improving reading systems accessibility. Currently these to resources are accessibility testing protocol on epubtest.org and W3C user agent accessibility guidelines.
I am good with adding such a brief information for reading systems accessibility in appendix of EPUB accessibility 1.1.

**Comment by murata2makoto** [comment]
I then like Matt's suggestion.  Incorporate the two bullets in the reading system specification.   Adding normative requirements that are not in the said scope of the accessibility  specification is not the right way to go forward, IMHO.

**Comment by mattgarrish** [comment]
It actually looks like we only have one requirement: that reading systems should conform to UAAG.

I don't believe we can require reading systems to pass a loosely defined test suite no matter which specification we put it in. I'm pretty sure we'll run afoul of W3C's requirements for normative references if we try to cite them normatively.

An informative note is likely all we could use to point devs to them.

**Comment by GeorgeKerscher** [comment]
I like the idea of adding accessibility requirements in the mainstream reading system specification. It further integrates accessibility into the day-to-day work of developers. This leaves the EPUB accessibility specification clearly focused on conformance and discovery of content. This also works well with the EU Directive, because they separate services (content) and products (reading systems and distribution systems).

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/publishing/groups/epub-wg/Meetings/Minutes/2021-04-08-epub-a11y#section2) on 2021-04-08

List of resolutions:

- ***[Resolution No. 2](https://www.w3.org/publishing/groups/epub-wg/Meetings/Minutes/2021-04-08-epub-a11y#resolution2):  add informative text for reading systems accessibility in the reading***

<details><summary><i>View the transcript</i></summary>

### 2. Where is the right place to put the language about RS a11y? 

_See github issue [#1608](https://github.com/w3c/epub-specs/issues/1608)._




**Avneesh Singh:** this used to be in the appendix  

**Matt Garrish:** this was considered out of scope for ISO  
… they wanted to focus spec on just the content aspect  
… so this went into the appendix in a11y 1.0  
… question is whether we should be incorporating this into RS spec  
… part of the problem is that we can't tie this to a specific test suite  
… there are also questions about CSS overrides, etc. and we can't really mandate a particular solution  
… but we can at least flesh out this recommendation language in the RS spec  

**Tzviya Siegman:** agreed, but I think this is an opportunity for us to work with SILVER  

**Avneesh Singh:** so for our current revision of the spec, are we leaning towards including this sort of language in the RS spec, or keep it in the a11y spec?  

**Wendy Reid:** agree with mgarrish  

**George Kerscher:** including it in RS spec would make it more obvious to RS developers that a11y is important  

> **Proposed resolution: add informative text for reading systems accessibility in the reading systems specifications/guidelines.** *(Avneesh Singh)*

> *Matt Garrish:* +1

> *Gregorio Pellegrino:* +1

> *Charles LaPierre:* +1

> *Wendy Reid:* +1

> *Ben Schroeter:* +1

> *Matthew Chan:* +1

> *Tzviya Siegman:* +1

> *George Kerscher:* +1

> ***Resolution #2: add informative text for reading systems accessibility in the reading***

</details>
